src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -65,15 +68,17 @@ protected override void Free()
 
         private NullableWalker(
             CSharpCompilation compilation,
-            Symbol member,
+            MethodSymbol member,
             BoundNode node,
             bool includeNonNullableWarnings)
             : base(compilation, member, node, new EmptyStructTypeCache(compilation, dev12CompilerCompatibility: false), trackUnassignments: false)
         {
             _sourceAssembly = ((object)member == null) ? null : (SourceAssemblySymbol)member.ContainingAssembly;
-            this._currentMethodOrLambda = member as MethodSymbol;
+            this._currentMethodOrLambda = member;
             _includeNonNullableWarnings = includeNonNullableWarnings;
             _binder = compilation.GetBinderFactory(node.SyntaxTree).GetBinder(node.Syntax);
```

Comment:
>compilation.GetBinderFactory(node.SyntaxTree).GetBinder(node.Syntax) [](start = 22, length = 68)

Are we interested specifically in a binder that is InMethodBinder here?

Reply:
Added `// PROTOTYPE` comment.

---
In reply to: [169826319](https://github.com/dotnet/roslyn/pull/24935#discussion_r169826319) [](ancestors = 169826319)